src/wallet/rpcwallet.cpp

```diff
@@ -4527,36 +4527,36 @@ static RPCHelpMan upgradewallet()
 #ifdef ENABLE_EXTERNAL_SIGNER
 static RPCHelpMan walletdisplayaddress()
 {
-    return RPCHelpMan{
-        "walletdisplayaddress",
-        "Display address on an external signer for verification.\n",
+    return RPCHelpMan{"walletdisplayaddress",
+        "\nDisplay address on an external signer for verification.\n",
         {
             {"address",     RPCArg::Type::STR, RPCArg::Optional::NO, /* default_val */ "", "bitcoin address to display"},
         },
         RPCResult{RPCResult::Type::NONE,"",""},
         RPCExamples{""},
-        [](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue {
-            std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
-            if (!wallet) return NullUniValue;
-            CWallet* const pwallet = wallet.get();
+    [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+{
```

Comment:
nit: For new code it would be good to use properly clang-formatted code. Any reason to make a change to move *away* from clang-formatted code?